### PR TITLE
subnetcalc 2.4.21

### PR DESCRIPTION
**Pull request by chenrui333**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

Also filed a question about the artifact location, https://github.com/dreibh/subnetcalc/issues/15

### CODE REVIEW

**Comment by iMichka** [comment]
It looks like releases are announced here: https://www.nntb.no/~dreibh/subnetcalc/#Download
Should we keep https://www.nntb.no/~dreibh/subnetcalc/ as `url` instead of using GitHub?

**Comment by chenrui333** [comment]
> It looks like releases are announced here: https://www.nntb.no/~dreibh/subnetcalc/#Download Should we keep https://www.nntb.no/~dreibh/subnetcalc/ as `url` instead of using GitHub?

looks like that artifact does not exist 🤦‍♂️ 
image

**Comment by chenrui333** [comment]
Will also update the artifact location issue with that info.

File: Formula/subnetcalc.rb [hunk 1]
```diff
@@ -25,6 +25,21 @@ def install
...
+      Broadcast     = not needed on Point-to-Point links
+      Wildcard Mask = 0.0.0.0
+      Hosts Bits    = 0
+      Max. Hosts    = 1   (2^0 - 0)
+      Host Range    = { 1.1.1.1 - 1.1.1.1 }
+      Properties    =
+         - 1.1.1.1 is a NETWORK address
+         - Class A
+      Performing reverse DNS lookup ...\r\e[KDNS Hostname  = one.one.one.one
+    EOS
```
Comment (carlocab) [inline Formula/subnetcalc.rb:42]: Seems a bit fragile.

**Review by carlocab — APPROVED** [review]

File: Formula/subnetcalc.rb (hunk 1 above)
Comment (chenrui333) [reply Formula/subnetcalc.rb:42]: indeed 😄 I will see how it goes.

**Pull request closed without merging**